ui/webui/src/components/review/ReviewConfiguration.jsx

```diff
@@ -70,17 +76,42 @@ export const ReviewDescriptionList = ({ children }) => {
     );
 };
 
-const DeviceRow = ({ name, data }) => {
+const DeviceRow = ({ name, data, requests }) => {
     const [isExpanded, setIsExpanded] = useState(false);
 
+    const renderRow = row => {
+        const iconColumn = row.reformat.v ? <CheckCircleIcon /> : null;
+        return {
+            props: { key: row["device-spec"].v },
+            columns: [
+                { title: row["device-spec"].v },
+                { title: row["format-type"].v },
+                { title: row["mount-point"].v },
+                { title: iconColumn },
+            ]
+        };
+    };
+
+    let imageRows = [];
+    if (requests !== null) {
+        imageRows = requests.filter(req => req["device-spec"].v.includes(name)).map(row => renderRow(row));
+    }
+
+    const columnTitles = [
+        { title: _("Partition") },
+        { title: _("Format type") },
+        { title: _("Mount") },
+        { title: _("Reformat") },
+    ];
```

Comment:
title is generally used when the column object is not a string. You can keep columnTitles as a list of strings.